src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -64,6 +64,14 @@
                     <CheckBox x:Uid="Show_name_suggestions"
                                   x:Name="Show_name_suggestions"
                                   Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Show_name_suggestions}" />
+                    
+                    <CheckBox x:Uid="Show_import_completion_items"
+                                  x:Name="Show_import_completion_items"
+                                  Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Show_import_completion_items}" 
```

Comment:
I see. it is because FAR take out references to generated files... so xaml can't reference string in resrouce file directly?

Reply:
looks like you could?
https://stackoverflow.com/questions/2919367/get-values-from-resx-files-in-xaml

so, not sure, why we have this static string type?

Reply:
I'm not sure either, just follow the pattern. @jasonmalinowski?